### DISCUSSION TITLE
Block Common Exploits?

### DISCUSSION THREAD

**Discussion by TechTucson** [NginxProxyManager/nginx-proxy-manager#1290] [Q&A]
Can you point me to some info on what this option does?

**Comment by chaptergy** [comment]
It mainly blocks specific GET / query parameters and also some user agents. 
When you enable the option [this config file](https://github.com/jc21/nginx-proxy-manager/blob/develop/docker/rootfs/etc/nginx/conf.d/include/block-exploits.conf) will be included in the nginx config.
